src/ethereum/osaka/vm/precompiled_contracts/ecrecover.py

```diff
@@ -54,7 +54,7 @@ def ecrecover(evm: Evm) -> None:
 
     try:
         public_key = secp256k1_recover(r, s, v - U256(27), message_hash)
-    except InvalidSignatureError:
+    except Exception:
```

Comment:
I feel like we could use specific exceptions here, given `secp256k1_recover` method raises a kind of InvalidSignatureError?